src/insertion_sort.py

```diff
@@ -0,0 +1,52 @@
+
+"""Bubble sort a list."""
+import time
+from random import randint
+
+
+def insert_sort(list):
```

Comment:
Choose a different variable name than `list` since `list` is a Python built-in `type` object